sdk/node/test/accessTokens.js

```diff
@@ -10,143 +10,147 @@ const expect = chai.expect
 
 const client = new chain.Client()
 
-let tokenId, clientTokenId, networkTokenId
-
-describe('Access tokens test', () => {
-
-  describe('Promise style', () => {
+function createToken() {
+  return client.accessTokens.create({
+    id: `token-${uuid.v4()}`
+  })
+}
 
-    describe('Access token', () => {
+describe('Access token', () => {
 
-      beforeEach(() => {
-        tokenId = `token-${uuid.v4()}`
-        return client.accessTokens.create({
-          type: 'client',
-          id: tokenId
-        })
-      })
-
-      it('creation successful', () => {
-        return client.accessTokens.create({
-          type: 'client',
-          id: `another-${tokenId}`
-        }).then(resp => expect(resp.token).not.to.be.empty)
-      })
+  it('creation successful', () => {
+    return client.accessTokens.create({
+      type: 'client',
```

Comment:
The access token type is deprecated, so it shouldn't be used unless we're explicitly testing the deprecated interface.